src/lib/isIdentityCard.js

```diff
@@ -1,6 +1,46 @@
 import assertString from './util/assertString';
+import isInt from './isInt';
 
 const validators = {
+  PL: (str) => {
+    assertString(str);
+
+    const weightOfDigits = {
+      1: 1,
+      2: 3,
+      3: 7,
+      4: 9,
+      5: 1,
+      6: 3,
+      7: 7,
+      8: 9,
+      9: 1,
+      10: 3,
+      11: 1,
+    };
+
+    if (str != null && str.length === 11 && isInt(str, { allow_leading_zeroes: true })) {
+      const digits = str.split('');
+      let sum = 0;
+
+      digits.forEach((digit, index) => {
+      // Ignored because handling else is unnecesary
+      /* istanbul ignore else */
+        if (index !== 10) {
+          sum += Number(digit) * weightOfDigits[index + 1];
+        }
+      });
```

Comment:
I'm pretty sure your `istanbul ignore else` is unneeded here. BTW i suggest using `Array.reduce` instead of forEach here
```suggestion
      const digits = str.split('').slice(0, -1);
      const sum = digits.reduce((acc, digit, index) =>
        acc + (Number(digit) * weightOfDigits[index + 1]), 0);
```

Reply:
I agree, so I have removed `Istanbul ignore else` and replaced Array.forEach with Array.reduce.